src/lib/services/types/evm/jsonRpc.ts

```diff
@@ -14,3 +14,16 @@ export const zJsonRpcResponse = z.object({
     .optional(),
 });
 export type JsonRpcResponse = z.infer<typeof zJsonRpcResponse>;
+
+export const zBatchJsonRpcResponse = z.union([
```

Comment:
Can we just use `export const zBatchJsonRpcResponse = z.array(zJsonRpcResponse);`?